src/main/java/io/stargate/sgv2/jsonapi/exception/FilterException.java

```diff
@@ -14,8 +14,10 @@ public FilterException(ErrorInstance errorInstance) {
   }
 
   public enum Code implements ErrorCode<FilterException> {
-    MULTIPLE_ID_FILTER,
-    FIELDS_LIMIT_VIOLATION;
+    // TODO, these Codes do not have a match in errors.yaml.
```

Comment:
Both to these are in the yaml file here https://github.com/stargate/data-api/blob/main/src/main/resources/errors.yaml#L123

there were added as place holders during the dev, now dont need them so they can be removed and the entries removed from errors.yaml

Reply:
thanks, removed